apps/tailwind-components/components/input/Group.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="mb-4">
+    <div v-if="label">
+      <label :for="id" class="text-title font-bold">{{ label }}</label>
+      <span class="text-disabled text-body-sm ml-3" v-show="required"
+        >Required
+      </span>
+    </div>
+    <div v-if="description" class="text-input-description text-body-sm">
```

Comment:
The description should also be written as a `<p>` element and linked to the input element using `aria-described by`.

Reply:
ok